python/test/unit/test_event_generators.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# test_event_generators.py
+
+import unittest
+import numpy as np
+
+import arbor as arb
+
+# to be able to run .py file from child directory
+import sys, os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../')))
+
+try:
+    import options
+except ModuleNotFoundError:
+    from test import options
+
+"""
+all tests for event generators (regular, explicit, poisson)
+"""
+
+class RegularSchedule(unittest.TestCase):
+    def test_default_regular_schedule(self):
+        rs = arb.regular_schedule()
```

Comment:
I don't know about testing default construction of these, because the test has to be rewritten if the default values are changed. You can convince me otherwise, if you think it is a good idea.

Reply:
Done. Set default values `tstart=None` and `tstop=None`.